lib/bulk_sql_delete.rb

```diff
@@ -0,0 +1,62 @@
+class BulkSqlDelete
```

Comment:
could we move this to `services`? most of the business logic is there, and `lib` is kind of unused